sdk/test-utils/perf/src/options.ts

```diff
@@ -143,6 +145,15 @@ export const defaultPerfOptions: PerfOptionDictionary<DefaultPerfOptions> = {
       "Set to true to use the Node worker_thread API when running tests across multiple CPUs. Set to false to use child_process (default).",
     defaultValue: false,
   },
+  profile: {
+    description:
+      "Set to true to profile the perf test. When set to true, `cpus` will be overriden to 1.",
+    defaultValue: false,
+  },
+  "profile-filepath": {
+    description: "Used as the artifact path",
+    defaultValue: undefined,
```

Comment:
here: could you simply compute the default output path here and then treat it as always-defined rather than handle the `undefined` case later?